src/modules/home/recommended-templates/RecommendedTemplates.tsx

```diff
@@ -7,15 +9,25 @@ import { templatesMock } from 'src/mocks/templates';
 import csx from './RecommendedTemplates.scss';
 
 export const RecommendedTemplates = () => {
+  const { push, location } = useHistory();
+
+  const goToDetails = useCallback(
+    (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+      const templateId = e.currentTarget.getAttribute('data-id');
+      push(`app/templates/all/${templateId}`);
+    },
+    [location]
```

Comment:
Placing here location is not neeed. This fn is using only data-id parameter so this can be removed